dev/com.ibm.ws.cdi.visibility_fat/publish/servers/cdi12BasicServer/server.xml

```diff
@@ -5,6 +5,7 @@
     <featureManager>
         <feature>osgiconsole-1.0</feature>
         <feature>cdi-1.2</feature>
+        <feature>jaxrs-2.0</feature>
```

Comment:
Why is this needed?

Reply:
I don't remember for certain, but I suspect it has something to do with this comment from the deleted WarLibsAccessWarBeansTest `//This test should be merged into BasicVisibilityTests when jaxrs-3.0 (EE9) is available`

jaxrs-3.0 is available, the tests were merged, and I suspect that during that process I hit an error that was resolved by explicitly listing jaxrs-2.0 in the features.

Reply:
Hmm, I can't see anything in the test that requires jaxrs at all.

I tried replacing jaxrs-2.0 with servlet-3.1 in the original test and it still passes so I don't think it was ever needed by the test.

In addition, it looks like the `cdi12WarLibsAccessWarServer` server can be removed.